assets/shaders/voxel_fragment/voxelize.frag.glsl

```diff
@@ -36,25 +36,25 @@ void discardIfOutsideAabb() {
 // by dividing them by voxelDimension.
 uvec4 calculateVoxelCoordinates() {
     uvec4 temp = uvec4(
-        gl_FragCoord.x,
-        gl_FragCoord.y,
-        voxelDimension * gl_FragCoord.z,
+        floor(gl_FragCoord.x),
+        floor(gl_FragCoord.y),
+        floor(float(voxelDimension) * gl_FragCoord.z),
```

Comment:
Isn't `gl_FragCoord.z` the depth buffer, so between 0 and 1?

Reply:
Yeah that why we are multiplicating between 0 and 1

Reply:
Yeah it makes sense, got confused :joy: